woocommerce/woocommerce-gutenberg-products-block
# Product Gallery: fix warning

## What

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/10836

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Open the Site Editor.
2. Edit the SIngle Product template
3. Add the Product Gallery.
4. Open the console.
5. Ensure that the error is not visible.



## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
